src/crud/workspace.py

```
@@ -0,0 +1,97 @@
+        logger.debug(f"Found existing workspace: {workspace.name}")
+        return existing_workspace
+
+    # Workspace doesn't exist, create a new one
+    honcho_workspace = models.Workspace(
+        name=workspace.name,
+        h_metadata=workspace.metadata,
+        configuration=workspace.configuration,
+    )
+    db.add(honcho_workspace)
+    await db.commit()
+    logger.info(f"Workspace created successfully: {workspace.name}")
```

Ruff (0.12.2)

Logging statement uses f-string

(G004)